ios/FluentUI.Demo/FluentUI.Demo/Demos/LeftNavDemoController.swift

```diff
@@ -182,7 +192,7 @@ class LeftNavMenuViewController: UIViewController {
             statusCellChild.leadingUIView = presence.imageView
             statusCellChild.backgroundColor = .systemBackground
             statusCellChild.onTapAction = {
-                self.setPresence(statusCell: statusCell, presence: presence)
+                self.setPresence(presence: presence)
```

Comment:
This will create a reference cycle if the self is not a weak reference here, right? (same for the actions you're setting throughout)